<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1516</width>
    <height>871</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1516</width>
    <height>871</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1516</width>
    <height>871</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>10</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

background-color: rgb(0, 0, 127);
color :rgb(255, 255, 255)
}
QPushButton:pressed{
background-color: rgb(255, 255, 255);
color :rgb(0, 0, 127)
}

</string>
    </property>
    <property name="text">
     <string>SEARCH</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>550</y>
      <width>221</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(9, 189, 238,20%);
 border-radius: 60px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1511</width>
      <height>871</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1511</width>
      <height>800</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1511</width>
      <height>871</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>851</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgba(9, 189, 238,20%);</string>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>190</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>CITY</string>
     </property>
     <property name="font">
      <font>
       <family>Berlin Sans FB</family>
       <pointsize>16</pointsize>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>PROVINCE</string>
     </property>
     <property name="font">
      <font>
       <family>Berlin Sans FB</family>
       <pointsize>16</pointsize>
       <underline>false</underline>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>POPULATION</string>
     </property>
     <property name="font">
      <font>
       <family>Berlin Sans FB</family>
       <pointsize>16</pointsize>
       <underline>false</underline>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>550</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>City</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>610</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Province</string>
    </property>
   </widget>
   <widget class="QLabel" name="degree">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>750</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>670</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Population                                 </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_USA">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>30</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

background-color: rgb(0, 0, 127);
color :rgb(255, 255, 255)
}
QPushButton:pressed{
background-color: rgb(255, 255, 255);
color :rgb(0, 0, 127)
}</string>
    </property>
    <property name="text">
     <string>USA</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_NL">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>80</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

background-color: rgb(0, 0, 127);
color :rgb(255, 255, 255)
}
QPushButton:pressed{
background-color: rgb(255, 255, 255);
color :rgb(0, 0, 127)
}

</string>
    </property>
    <property name="text">
     <string>NL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TR">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>130</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	

background-color: rgb(0, 0, 127);
color :rgb(255, 255, 255)
}
QPushButton:pressed{
background-color: rgb(255, 255, 255);
color :rgb(0, 0, 127)
}</string>
    </property>
    <property name="text">
     <string>TR</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgba(255, 255, 255,20%);</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhHiddenText</set>
    </property>
    <property name="text">
     <string notr="true">City Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>250</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="description">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>470</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhPreferUppercase</set>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="cityname_gui">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>550</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="provincename_gui">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>610</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="population_gui">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>670</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="code">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>760</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="code_2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>60</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="degree_2">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>760</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>°C</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="revert_button">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius:30px;
	color: rgb(0, 0, 0);

background-color: rgb(0, 0, 0);
color :rgb(255, 255, 255)
}
QPushButton:pressed{
background-color: rgb(255, 255, 255);
color :rgb(0, 0, 127)
}</string>
    </property>
    <property name="text">
     <string>Revert</string>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>tableWidget</zorder>
   <zorder>label</zorder>
   <zorder>pushButton</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>degree</zorder>
   <zorder>label_6</zorder>
   <zorder>pushButton_USA</zorder>
   <zorder>pushButton_NL</zorder>
   <zorder>pushButton_TR</zorder>
   <zorder>lineEdit</zorder>
   <zorder>description</zorder>
   <zorder>cityname_gui</zorder>
   <zorder>provincename_gui</zorder>
   <zorder>population_gui</zorder>
   <zorder>code</zorder>
   <zorder>degree_2</zorder>
   <zorder>code_2</zorder>
   <zorder>label_7</zorder>
   <zorder>revert_button</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
